src/book/02-system/03-tools/02-nubook.mdx

```diff
@@ -26,9 +26,15 @@ You'll need [Git](https://git-scm.com/), [Node.js](https://nodejs.org/en/) and [
    sudo apt install git
    ```
 
-2. Install the latest version of Node.js by following the instructions [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) (pick the latest version).
+2. Install Node.js version 14 by following the instructions [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) and running the commands under "Node.js v14.x".
 
-3. Install the latest version of Yarn by following the instructions [here](https://classic.yarnpkg.com/en/docs/install#debian-stable).
+3. Install the latest version of Yarn
```

Comment:
You should just be able to do `npm -g install yarn` after installing Node 🤔 is there a reason you are installing yarn directly from source? Installing a specific version of Node and then the latest version of yarn could potentially introduce a conflict whereas npm will manage the version properly

Reply:
API has slightly changed but yeah the way I would do this would be
`curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash -
sudo apt-get install -y nodejs
npm install yarn --location=global`

Reply:
Might be worth updating the instructions here as well https://github.com/NUbots/NUbook/blob/main/src/book/03-guides/01-main-codebase/01-getting-started.mdx

Reply:
`npm -g install yarn` wasn't working for me for some reason and was able to with `npm install --global yarn
` I will go through the setup again tonight and verify which commands work / don't.

Reply:
The npm one may need to run as sudo

Reply:
Okay, so after I went from Pop_OS version 22.0x -> 22.0x, the original installation works.

Reply:
Whatever is currently there written works, provided you install Node 16 over 18. So I will update it.